docs/source/usage/plugins/phaseSpace.rst

```diff
@@ -37,47 +37,61 @@ Option                                 Usage
 Output
 ^^^^^^
 
-Values are given as *charge density* per phase space bin.
-In order to scale to a simpler *charge of particles* per :math:`\mathrm{d}r_i` and :math:`\mathrm{d}p_i` -bin multiply by the cell volume:
+The histograms are stored in ``.hdf5`` files in the ``simOutput/...`` directory.
 
-.. code:: python
+...
 
-   ps = ... # get data set from h5py
+Values are given as *charge density* per phase space bin.
+In order to scale to a simpler *charge of particles* per :math:`\mathrm{d}r_i` and :math:`\mathrm{d}p_i` -bin multiply by the cell volume ``dV``.
 
-   # 3D3V example
-   dV = ps.attrs['dV'] * ps.attrs['dr_unit']**3
-   charge_per_bin = ps[:,:] * ps.attrs['sim_unit'] * dV
+Analysis
+^^^^^^^^
 
-Out-of-Range Behavior
-^^^^^^^^^^^^^^^^^^^^^
+The easiest way is to load the data in Python:
 
-Particles that are *not* in the range of ``<ValL>``/``<ValR>`` get automatically mapped to the lowest/highest bin respectively.
-Take care about that when setting your range and during analysis of the results.
+.. code:: python
 
-Spatial Offset
-^^^^^^^^^^^^^^
+   from picongpu.plugins.phase_space import PhaseSpace
+   import matplotlib.pyplot as plt
+   from matplotlib.colors import LogNorm
+   import numpy as np
 
-The spatial extent of the output might change due to a moving window.
-Additional attributes are provided to retrieve that spatial information:
 
-See this python example:
+   # load data
+   phase_space = PhaseSpace('/home/axel/runs/foil_001')
+   e_ps, e_ps_meta = phase_space.get('e', ps='ypy', iteration=1000)
 
-.. code:: python
+   # unit conversion from SI
+   mu = 1.e6  # meters to microns
+   e_mc_r = 3.6618e21  # electrons: kg * m / s to beta * gamma
 
-   ps = ... # get data set from h5py
+   # plotting
+   plt.imshow(
+       np.abs(e_ps) * e_ps_meta.dV,
+       extent = e_ps_meta.extent * [mu, mu, e_mc_r, e_mc_r],
+       interpolation = 'nearest',
+       aspect = 'auto',
+       origin='lower',
+       norm = LogNorm()
+   )
 
-   mv_start = ps.attrs['movingWindowOffset']
-   mv_end = mv_start + ps.attrs['movingWindowSize']
-   spatial_offset = ps.attrs['_global_start'][1] # 2D data set: 0 (r_i); 1 (p_i)
+   # annotations
+   cbar = plt.colorbar()
+   cbar.set_label(r'$Q / \mathrm{d}r \mathrm{d}p$ [$C s kg^{-1} m^{-2}$]')
 
-   dr = ps.attrs['dr'] * ps.attrs['dr_unit']
+   ax = plt.gca()
+   ax.set_xlabel(r'${0}$ [$\mu m$]'.format(e_ps_meta.r))
```

Comment:
QUESTION:
why do you write `{0}` instead of `{}`?

Reply:
this is just making it python 2.6 compatible. not really important

Reply:
Ahh - thanks I didn't know that

Reply:
me neither, learned it while contributing to spack :)